.maintain/frame-weight-template-clippy.hbs

```diff
@@ -0,0 +1,104 @@
+//! Autogenerated weights for {{pallet}}
```

Comment:
I guess this file can just be called `frame-weight-template.hbs` and override the default. We can always get the default one if we wanted but having a different file might be confusing to other team members if they're not aware they should use this one.